CHANGES/181.feature

```diff
@@ -0,0 +1,35 @@
+Added an interactive-shell mode to pulp-cli.
+
+Starting the CLI with "pulp shell" drops you into the shell:
+```
+(pulp) [vagrant@pulp3 ~]$ pulp shell
+Starting Pulp3 CLI...
+pulp-cli> help
+
+Documented commands (type help <topic>):
+========================================
+access-policy  config     export    group      orphans     rpm     task
+ansible        container  exporter  importer   python      show    user
+artifact       debug      file      migration  repository  status  worker
+
+Undocumented commands:
+======================
+exit  help  quit
+
+pulp-cli> status
+{
+    ...
+}
+
+pulp-cli>
+(pulp) [vagrant@pulp3 ~]$
+```
+
+Issuing the command with arguments works as it does currently:
+```
+(pulp) [vagrant@pulp3 ~]$ pulp status
+{
+    ...
+}
+(pulp) [vagrant@pulp3 ~]$
+```
```

Comment:
This is probably too much detail for the changelog. I'd recommend creating a shell doc in the docs folder.

Reply:
I agree. However, it's unclear to me which one is the pr we want to use here, and this file is originally from the other one.

Reply:
We def want to use @gerrod3 's PR, it answers the "run as pulp shell" request from my PR. I'm going to close mine and we'll continue on this one.

I think moving the example to its own doc page is a fine idea.

Reply:
+1 Moved!